app/frontend/src/features/profile/edit/EditHostingPreference.tsx

```diff
@@ -255,7 +255,7 @@ export default function HostingPreferenceForm() {
             name="smokingAllowed"
             render={({ onChange, value }) => (
               <Select
-                onChange={onChange}
+                onChange={(event) => onChange(event.target.value)}
```

Comment:
How was this working before if `onChange` needs the value explicitly here? 🤔

Reply:
No idea. Are we sure it's not broken? Seems like it should be broken.